crate/pages/jobs/java-software-engineer.html

```diff
@@ -1,29 +1,19 @@
-title: Java Software Engineer
-description:
-role: Software Engineering
-location: Berlin/Germany, Dornbirn/Austria
-type: Full-time
-updated: Dec 04, 2014
+title: Java Software Engineer description: role: Software Engineering location: Berlin/Germany, Dornbirn/Austria type: Full-time updated: Dec 04, 2014
```

Comment:
That's my editor, I have a set up optimised for 'proper' markdown and it tends not to like this 'pseudo' markdown and automatically fix it. Usually I remember to correct it afterwards, sorry, will update before merging.